blocks/library/embed/index.js

```diff
@@ -144,9 +146,18 @@ function getEmbedBlockSettings( { title, icon, category = 'embed', transforms, k
 					)
 				);
 
+				const inspectorControls = focus && (
+					<InspectorControls key="inspector">
+						<BlockDescription>
+							<p>{ __( 'The Embed block allows you to easily add videos, images, tweets, audio, and other content to your post or page.' ) }</p>
+						</BlockDescription>
+					</InspectorControls>
+				);
+
 				if ( fetching ) {
 					return [
 						controls,
+						inspectorControls,
```

Comment:
Minor: It looks like the `controls` and `inspectorControls` are used in the exact same places, do you think we should merge them under the same array variable `controls` ?

Reply:
I like that idea. `controls` would become an array and then you could use destructuring to apply in the return statement as follows:
```js
return [
    ...controls,
    <ComponentName />
];
```

Reply:
I guess it works without destructuring too

Reply:
Is this what you mean?

![2017-10-12_00-10-08](https://user-images.githubusercontent.com/1571635/31483563-c4cee8aa-aee1-11e7-9347-231aa248f67d.png)

I see PHPStorm throwing a warning:

![2017-10-12_00-10-21](https://user-images.githubusercontent.com/1571635/31483569-ccd3e3fc-aee1-11e7-9f53-73a7c4327b95.png)

Reply:
It should probably be something like

```js

const controls = [
  focus && ( 
    // ...
  ),
  focus && ( 
    // ...
  ),
];
```

Brackets instead of parenthesis.

Reply:
Derp. @youknowriad - it's late here haha. I've updated it.